packages/types/src/CharacterBonus.ts

```diff
@@ -13,6 +13,10 @@ class CharacterBonus extends Bonus {
   }
 
   public matchesCellTileScore(config: Config, cell: Cell): boolean {
+    if(typeof this.score === 'undefined') {
```

Comment:
There is a code formatting error here:

```suggestion
    if (typeof this.score === 'undefined') {
```